cmd/config/subcommand/project/project_config.go

```diff
@@ -11,7 +11,33 @@ var (
 	}
 )
 
-// Config
+// Config holds the flag for get project
 type Config struct {
 	Filter filters.Filters `json:"filter" pflag:","`
 }
+
+type Definition struct {
+	ID          string            `yaml:"id"`
+	Name        string            `yaml:"name"`
+	Description string            `yaml:"description"`
+	Labels      map[string]string `yaml:"labels"`
+}
```

Comment:
Any reason we have Definition and same properties part of the ConfigProject.
Instead the definition can be embedded inside ConfigProject to avoid duplication of attributes